fhir-search/src/main/java/com/ibm/fhir/search/parameters/QueryParameter.java

```diff
@@ -95,78 +93,73 @@ public String toString() {
         if (modifier != null) {
             buffer.append("modifier: ");
             buffer.append(modifier.value());
-            buffer.append(SearchConstants.NL);
+            buffer.append(NL);
         }
 
         if (modifierResourceTypeName != null) {
             buffer.append("modifierTypeResourceName: ");
             buffer.append(modifierResourceTypeName);
-            buffer.append(SearchConstants.NL);
+            buffer.append(NL);
         }
 
-        boolean composite = isComposite();
-        buffer.append("composite: ");
-        buffer.append(composite);
-        buffer.append(SearchConstants.NL);
-
         boolean chained = isChained();
         buffer.append("chained: ");
         buffer.append(chained);
-        buffer.append(SearchConstants.NL);
+        buffer.append(NL);
 
         boolean inclusionCriteria = this.isInclusionCriteria();
         buffer.append("inclusionCriteria: ");
         buffer.append(inclusionCriteria);
-        buffer.append(SearchConstants.NL);
+        buffer.append(NL);
 
-        List<ParameterValue> values = getValues();
-        for (ParameterValue value : values) {
+        List<QueryParameterValue> values = getValues();
+        for (QueryParameterValue value : values) {
             Prefix prefix = value.getPrefix();
             if (prefix != null) {
                 buffer.append("    prefix: ");
                 buffer.append(prefix.value());
-                buffer.append(SearchConstants.NL);
+                buffer.append(NL);
             }
 
             String valueString = value.getValueString();
             if (valueString != null) {
                 buffer.append("    valueString: " + valueString);
-                buffer.append(SearchConstants.NL);
+                buffer.append(NL);
             }
 
             DateTime valueDate = value.getValueDate();
             if (valueDate != null) {
                 buffer.append("    valueDate: " + valueDate.getValue());
-                buffer.append(SearchConstants.NL);
+                buffer.append(NL);
             }
 
             BigDecimal valueNumber = value.getValueNumber();
             if (valueNumber != null) {
                 buffer.append("    valueNumber: " + valueNumber.toPlainString());
-                buffer.append(SearchConstants.NL);
+                buffer.append(NL);
             }
 
             String valueSystem = value.getValueSystem();
             if (valueSystem != null) {
                 buffer.append("    valueSystem: " + valueSystem);
-                buffer.append(SearchConstants.NL);
+                buffer.append(NL);
             }
 
             String valueCode = value.getValueCode();
             if (valueCode != null) {
                 buffer.append("    valueCode: " + valueCode);
```

Comment:
we should probably split this into two appends. 
assemble string1 and string 2 into string 3
append string 3

Reply:
Not sure how much this matters; the one thing I did is add a static import for `NL`.  And since I did that for one spot, I went through and removed the unneccessary class qualifier everywhere else...the rest was already here.

Reply:
didn't do this one, but can if its important to you

Reply:
no worries. this is a-ok